src/ColorWrapper.cpp

```diff
@@ -0,0 +1,110 @@
+#include <Gosu/Color.hpp>
+#include <Gosu/Color.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+Gosu_Color* Gosu_Color_create(unsigned argb)
+{
+  return reinterpret_cast<Gosu_Color*>( new Gosu::Color(argb) );
```

Comment:
Since Color is really just an `unsigned` internally, I think it would be good if we could export it as such, and avoid extra allocations for these tiny objects.